docs/docs/15-concepts.md

```diff
@@ -464,11 +476,16 @@ A single `Freight` resource references one or more versioned artifacts, such as:
 
 * Helm charts (from chart repositories)
 
-A `Freight` resource's `id` field _and_ `metadata.name` field are both (for now)
-set to the same value, which is a SHA1 hash of a canonical representation of the
-artifacts referenced by the `Freight` resource. (This is enforced by an
-admission webhook.) The `id` and `metadata.name` fields, therefore, are both
-"fingerprints," deterministically derived from the `Freight`'s contents.
+A `Freight` resource's `metadata.name` field is a SHA1 hash of a canonical
+representation of the artifacts referenced by the `Freight` resource. (This is
+enforced by an admission webhook.) The `metadata.name` field is therefore a
+"fingerprint", deterministically derived from the `Freight`'s contents.
+
+To provide a human-readable identifier for a `Freight` resource, a `Freight`
+resource has an `alias` field and `kargo.akuity.io/alias` label. This alias is
+a human-readable string that is unique within the `Project` to which the
+`Freight` belongs. While it can be set manually (and changed), by default,
+Kargo will automatically generate a unique alias for each `Freight` resource.
```

Comment:
I think we may need an info box or something to mention that the alias field is a convenience (so no one ever needs to go looking for the label) and maybe something about precedence:

- If field is specified, webhook syncs the label value to the field value
- If field is unspecified, webhook syncs the field field value to the label value

fwiw, that means removing an alias _entirely_ requires you to clear both of these, but I think that is something people will rarely, if ever, do. We probably don't need to call that out. 🤷

Reply:
As this is a concepts page, I was worried about putting the information in the wrong place. To solve this, I have added it to the section about updating aliases and added references to this page instead.

I think this would potentially be good to do in other places as well in the future, to ensure we reduce the number of places where things are documented twice.